skt/executable.py

```diff
@@ -502,21 +502,29 @@ def setup_parser():
     parser_merge.add_argument(
         "--patchlist",
         type=str,
-        nargs="+",
-        help="Paths to each local patch to apply (space delimited)"
+        action='append',
+        help=(
+            "Paths to each local patch to apply "
+            "(use multiple times for multiple patches)"
```

Comment:
Shouldn't the name of the option be changed too, since the list shouldn't be provided anymore?

Reply:
Right. I'll fix that.